docs/data/sql_functions.yml

```diff
@@ -525,7 +525,11 @@ conditional:
     description: Returns true if all characters in string are digit, otherwise false.
   - table: BOOLEAN.?(VALUE1, VALUE2)
     description: Returns VALUE1 if BOOLEAN evaluates to TRUE; returns VALUE2 otherwise. E.g., (42 > 5).?('A', 'B') returns "A".
-   
+  - sql: GREATEST(value1[, value2]*)
+    description: Returns the greatest VALUE if no VALUE is NULL. Returns NULL if at least one VALUE is NULL.
```

Comment:
I think the "VALUE"s here (and below) shouldn't be uppercased since they aren't in the syntax above.

Reply:
Thanks @Airblader for the review, I've updated the description.